### DISCUSSION TITLE
Tile servers with 512 pixel map tiles

### DISCUSSION THREAD

**Discussion by activityworkshop** [osmandapp/OsmAnd#13565] [Maps Related]
Hello OsmAnd team!

I was recently given an interesting URL as an example of 512-by-512 pixel map tiles rather than the usual 256-by-256 pixels, here:
https://tile.osmand.net/hd/10/547/319.png
This larger rendering looks like could be really useful (especially for those with small pixels on their screens) and it's the first such source I seen with general coverage (the only other one I've found is restricted to a single country, so not generally useful for everybody).

I'm pretty sure that you don't want just any map client using your tile server, and I assume that other clients directly getting these tiles via https would come under "You may not integrate [the service] within a service of your own" in the terms, is that correct?

In that case, maybe I could broaden the question to ask about _other_ tile servers which you use, and if any of them include extra-large rendering and/or 512-pixel tile sizes (I guess they usually go together but maybe not always?).  Do you know of any such servers, preferably worldwide and freely available?

Context: this is in the context of a PC-based (not mobile) map client application called GpsPrune.  Some users could benefit from larger renderings but don't want to render their own tiles themselves.  And because the application is free, I don't think Thunderforest is a possible option either.  And I really want to avoid scaling the tiles up to make the texts readable, I'd like to display them all 1:1.  Ideally osm would just provide regular 256-by-256 tiles in a different style using much larger fonts and features (called "senior" style or "accessible" style) but I haven't found this.  Am I missing something?

Thank you for any advice / thoughts!

**Comment by vshcherb** [comment]
https://tile.osmand.net/df/10/547/319.png

**Comment by mariush444** [comment]
https://services.arcgisonline.com/arcgis/rest/services/ESRI_Imagery_World_2D/MapServer/tile/10/319/547

**Comment by activityworkshop** [comment]
Thanks for both those links!
Yes, the "df" tiles look like the regular tiles, and the problem is that the writing is quite small if the user's screen has small pixels (or if the user's eyesight is no longer what it was).  The rendering used to make the "hd" tiles produces much more readable text for those cases.  You can see the difference here:
![tile_examples](https://user-images.githubusercontent.com/11003522/150575158-bffdace6-abab-4c21-9a1a-9c49d6be52ca.png)

Basically I'd like to know:
 * Do you know of any 256-pixel tiles (which I am allowed to use over https) using such larger renderings like these "hd" tiles?
 * If not, do you know the osmand policy for whether I am allowed to use the "hd" tiles over https from a PC application?
 
 Thanks also for the arcgisonline link, that's a very interesting aerial set!  However right now I'm more interested in maps than aerial images.

**Reply by vshcherb** [reply]
Actually we have no policy so far. So I think nothing changes and we've got a big overload on the servers, you're fine to use it under fair policy. If something changes, we will let you know. 
Yes we've increased text size because it better fits mobile for HD but it's probably not needed for pure web

**Reply by activityworkshop** [reply]
That's very generous, thank you!
Nothing will change in the short term because I can't handle 512-pixel tiles yet, but maybe at some point in the future I'll see if I can start to use them.  Thanks very much!
